sdk/eventhub/azure-eventhub/azure/eventhub/_consumer.py

```diff
@@ -183,8 +184,8 @@ def _next_message_in_buffer(self):
         # pylint:disable=protected-access
         message = self._message_buffer.popleft()
         event_data = EventData._from_message(message)
-        if self._amqp_transport != PyamqpTransport:
-            event_data._uamqp_message == message    # pylint: disable=pointless-statement
+        if self._amqp_transport.KIND != "pyamqp":
+            event_data._uamqp_message = message    # pylint: disable=pointless-statement
```

Comment:
I think the pylint disable can be removed now that you've fixed the `==`
```suggestion
            event_data._uamqp_message = message
```

Reply:
oo yep makes sense!